src/assets/scripts/client/aircraft/FlightManagementSystem/Fms.js

```diff
@@ -514,6 +515,70 @@ export default class Fms {
         return this._routeModel.getFullRouteStringWithoutAirportsWithSpaces();
     }
 
+    /**
+     * Return the next waypoint having an #altitudeMaximum equal to or less than the specified value
+     *
+     * This is helpful to see only future waypoints for which a particular altitude is
+     * considered NOT to be compliant. This allows us to focus which waypoints the aircraft
+     * will need to adjust altitude for, even if they are many waypoints in the future.
+     *
+     * @for Fms
+     * @method getNextWaypointWithMaximumAltitudeAtOrBelow
+     * @param altitude {number}
+     * @return {WaypointModel}
+     */
+    getNextWaypointWithMaximumAltitudeAtOrBelow(altitude) {
+        return _find(this.waypoints, (waypointModel) => waypointModel.hasMaximumAltitudeAtOrBelow(altitude));
+    }
+
+    /**
+     * Return the next waypoint having an #altitudeMinimum equal to or greater than the specified value
+     *
+     * This is helpful to see only future waypoints for which a particular altitude is
+     * considered NOT to be compliant. This allows us to focus which waypoints the aircraft
+     * will need to adjust altitude for, even if they are many waypoints in the future.
+     *
+     * @for Fms
+     * @method getNextWaypointWithMinimumAltitudeAtOrAbove
+     * @param altitude {number}
+     * @return {WaypointModel}
+     */
+    getNextWaypointWithMinimumAltitudeAtOrAbove(altitude) {
```

Comment:
same here, use `find` in the name instead of `get`

Reply:
done